ISSUE_TEMPLATE.md

```diff
@@ -1,100 +1,123 @@
-###### the appropriate gnu-efi source.
-###### Please confirm this as the origin your shim.
-`[your text here]`
-
-###### What's the justification that this really does need to be signed for the whole world to be able to boot it:
+-------------------------------------------------------------------------------
+### What organization or people are asking to have this signed?
+-------------------------------------------------------------------------------
 `[your text here]`
 
-###### How do you manage and protect the keys used in your SHIM?
+-------------------------------------------------------------------------------
+### What product or service is this for?
+-------------------------------------------------------------------------------
 `[your text here]`
 
-###### Do you use EV certificates as embedded certificates in the SHIM?
+-------------------------------------------------------------------------------
+### How do you manage and protect the keys used in your SHIM?
+-------------------------------------------------------------------------------
 `[your text here]`
 
-###### If you use new vendor_db functionality, are any hashes allow-listed, and if yes: for what binaries ?
+-------------------------------------------------------------------------------
+### Do you use EV certificates as embedded certificates in the SHIM?
+-------------------------------------------------------------------------------
 `[your text here]`
 
-###### Is kernel upstream commit 75b0cea7bf307f362057cc778efe89af4c615354 present in your kernel, if you boot chain includes a Linux kernel ?
+-------------------------------------------------------------------------------
+### If you use new vendor_db functionality, are any hashes allow-listed?
+### If yes: for what binaries?
+-------------------------------------------------------------------------------
 `[your text here]`
 
-###### if SHIM is loading GRUB2 bootloader, are CVEs CVE-2020-14372,
-###### CVE-2020-25632, CVE-2020-25647, CVE-2020-27749, CVE-2020-27779,
-###### CVE-2021-20225, CVE-2021-20233, CVE-2020-10713, CVE-2020-14308,
-###### CVE-2020-14309, CVE-2020-14310, CVE-2020-14311, CVE-2020-15705,
-###### ( July 2020 grub2 CVE list + March 2021 grub2 CVE list )
-###### and if you are shipping the shim_lock module CVE-2021-3418
-###### fixed ?
+-------------------------------------------------------------------------------
+### If your boot chain of trust includes a Linux kernel:
+### Is upstream commit [1957a85b0032a81e6482ca4aab883643b8dae06e "efi: Restrict efivar_ssdt_load when the kernel is locked down"](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=1957a85b0032a81e6482ca4aab883643b8dae06e) applied?
+### Is upstream commit [75b0cea7bf307f362057cc778efe89af4c615354 "ACPI: configfs: Disallow loading ACPI tables when locked down"](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=75b0cea7bf307f362057cc778efe89af4c615354) applied?
```

Comment:
When this question was introduced it only asked for one commit here, but two commits in the README. I don't know why, so I've put both here. That might be wrong.